jenkins/Jenkinsfile

```diff
@@ -0,0 +1,109 @@
+def secrets = [
+  [path: 'secret/jenkins/github', engineVersion: 2, secretValues: [
+    [envVar: 'PRIVATE_REPO_GITHUB_TOKEN', vaultKey: 'private_token']]],
+  [path: 'secret/jenkins/npm', engineVersion: 2, secretValues: [
+    [envVar: 'NPM_TOKEN', vaultKey: 'token']]],
+]
+
+def configuration = [vaultUrl: "${env.VAULT_URL}",  vaultCredentialId: 'vault-app-role', engineVersion: 2]
+
+pipeline {
+  agent {
+    kubernetes {
+      label 'common-ui'
+      defaultContainer 'jnlp'
+      yamlFile 'jenkins/build-pod.yaml'
+    }
+  }
+  
+  environment {
+    MAIN_BRANCH = 'master'
+    WORKSPACE = "${env.WORKSPACE}"
+    BRANCH = "${env.BRANCH}"
+    PROJECT = "cloudify-ui-common"
+  }
+
+  options {
+    checkoutToSubdirectory('cloudify-ui-common')
```

Comment:
Possibly we can use `${env.PROJECT}` here as in stages?

Reply:
It will not work, env variables for stages only, you can trust options will init correctly from env